tools/start_all_tmux.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is used to start all of the microservices
+# for the Lucida project by creating a tmux window for each
+# server in the background.
+# Example calls:
+# $./start_all_tmux.sh secure
+# $./start_all_tmux.sh
+# To attach to the tmux session
+# use the following command:
+# $tmux a -t lucida
+
+SESSION_NAME="lucida"
+
+# Check to see if we should run on http/ws (non-secure) or https/wss (secure)
+if [ "$1" == "secure" ]; then
+    echo "Enabling secure host"
+    # Getting the host IP address
+    export ASR_ADDR_PORT="wss://$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}'):8081"
+    export SECURE_HOST=true
+
+    # Generate self-signed certificates
+    cd $(pwd)/../lucida/commandcenter/
+    chmod +x gen_certs.sh
+    ./gen_certs.sh
+    cd $(pwd)/../../tools
+else
+    echo "Enabling non-secure host"
+    export ASR_ADDR_PORT="ws://localhost:8081"
+    unset SECURE_HOST
+fi
+
+declare -a commandcenter=("CMD" "$(pwd)/../lucida/commandcenter/")
+declare -a questionanswering=("QA" "$(pwd)/../lucida/questionanswering/OpenEphyra/")
+declare -a imagematching=("IMM" "$(pwd)/../lucida/imagematching/opencv_imm/")
+declare -a calendar=("CA" "$(pwd)/../lucida/calendar/")
+declare -a speechrecognition=("ASR" "$(pwd)/../lucida/speechrecognition/kaldi_gstreamer_asr/")
+declare -a imageclassification=("IMC" "$(pwd)/../lucida/djinntonic/imc/")
+declare -a digitrecognition=("DIG" "$(pwd)/../lucida/djinntonic/dig/")
+declare -a facerecognition=("FACE" "$(pwd)/../lucida/djinntonic/face")
+
+declare -a services=(
+    commandcenter
+    questionanswering
+    imagematching
+    calendar
+    speechrecognition
+    imageclassification
+    digitrecognition
+    facerecognition)
+
+# Create the session
+tmux new-session -s ${SESSION_NAME} -n vim -d
```

Comment:
I've gotten as far seeing different options out there. This article has a good comparison of different options (http://centos-vn.blogspot.com/2014/06/daemon-showdown-upstart-vs-runit-vs.html). I'll need to do more research about what would be best for our situation.